sql/updates/world/3.3.5/9999_99_99_99_world.sql

```diff
@@ -0,0 +1,3 @@
+INSERT INTO `spell_script_names` VALUES
+(52479, 'spell_gift_of_the_harvester');
+UPDATE `gameobject_template` SET `Data3`=0,`AIName`='',`ScriptName`='gift_of_the_harvesterAI' WHERE `entry`=190769;
```

Comment:
why data3=0?

Reply:
Because spell will be casted early, but animation is missing.

Reply:
Animations are not implemented for al most all gobs type, better to keep the original value and wait for a generic solution.

Reply:
Just as a footnote: Aokromes always used to tell me that `gameobject_template`.`Data*` are sniffed data and should not be changed.

Reply:
Here the sequence ussed by the gob (and probably any trap) https://gist.github.com/Killyana/05753c48ad36b49905924f4ed1811cca
01:59:23 - the gob is summoned
01:59:24 -  the spell 52479  is casted by the gob
01:59:24 - SMSG_GAMEOBJECT_CUSTOM_ANIM
01:59:26 - SMSG_GAMEOBJECT_DESPAWN_ANIM
01:59:26 - SMSG_DESTROY_OBJECT
We have already an issue https://github.com/TrinityCore/TrinityCore/issues/23109 about this.
So no need to change the data or use extra code to activate the gob animation.

Reply:
Well, then I am going to revert gob changes (spell script is still welcome I believe).